dashboard/lib/school_info_deduplicator.rb

```diff
@@ -28,26 +28,12 @@ def get_duplicate_school_info(school_info_attr)
   def process_school_info_attributes(school_info_attr)
     attr = school_info_attr.symbolize_keys
 
-    # Names of state and zip fields change between form fields and SchoolInfo class
```

Comment:
Why can this stuff be removed now?

Reply:
it started with removing line 33, which is just translating the shape of the request. since `buildSchoolData` sends `zip` and not `school_zip` this translation was no longer necessary. then since we don't care about district, address, etc for non nces schools, I decided to clean this up as well. of course that meant more tests to update... I think it's still a good change, but I could wait on it and do it in the school_info cleanup ticket if that's preferred

Reply:
Given some things are still using the old school info inputs, do they not still need this code path?

Reply:
yeah, that's a fair point! I'll hold off on this change and note it for the cleanup pr